client/consumables.lua

```diff
@@ -1,25 +1,14 @@
 -- Variables
 
 local QBCore = exports['qb-core']:GetCoreObject()
-local alcoholCount = 0
-local ParachuteEquiped = false
-local currentVest = nil
-local currentVestTexture = nil
-local healing = false
+local alcoholCount, ParachuteEquiped, currentVest, currentVestTexture, healing, SmokingWeed, RelieveCount = 0, false, nil, nil, false, false, 0
 
 -- Functions
 
-local function loadAnimDict(dict)
-    if HasAnimDictLoaded(dict) then return end
-    RequestAnimDict(dict)
-    while not HasAnimDictLoaded(dict) do
-        Wait(10)
-    end
-end
-
 local function EquipParachuteAnim()
-    loadAnimDict("clothingshirt")
-    TaskPlayAnim(PlayerPedId(), "clothingshirt", "try_shirt_positive_d", 8.0, 1.0, -1, 49, 0, false, false, false)
+    local hasLoaded = lib.requestAnimDict('clothingshirt', 1000)
```

Comment:
Leave out the timeout, we don't want to wait that long, the default is fine

Reply:
You don't need to wait that long, as soon as it loads it will stop and start the animation regardless of how long that timer is but okay.

Reply:
I know you don't have to wait that long if it loads instantly, but if it doesn't, the intervals of it waiting are too high, which people with worse PCs usually will experience

Reply:
Yeah its changed